src/operator/tensor/la_op.cc

```diff
@@ -420,5 +421,134 @@ NNVM_REGISTER_OP(_backward_linalg_sumlogdiag)
 .set_attr<nnvm::TIsBackward>("TIsBackward", true)
 .set_attr<FCompute>("FCompute<cpu>", LaOpBackward<cpu, 2, 2, 2, 1, sumlogdiag_backward>);
 
+NNVM_REGISTER_OP(linalg_syrk)
```

Comment:
Initial underscore followed by alias (as well as in the other new operators)

Reply:
OK, will do this